google-cloud-storage/pom.xml

```diff
@@ -156,6 +156,11 @@
       <artifactId>easymock</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
```

Comment:
I think `mockito-core` is more than enough for these test cases.

Reply:
okay, sounds good.